pandas/core/dtypes/missing.py

```diff
@@ -1,8 +1,11 @@
 """
 missing types & inference
 """
+from __future__ import annotations
```

Comment:
Is this really needed?  What piece of code is requiring this?

Reply:
Needed for `result: np.ndarray | ExtensionArraySupportsAnyAll`